tasks/main.yml

```
@@ -1,4 +1,20 @@
 ---
+
+- name: Ensure nix user exists
+  user:
```

I'm pretty sure that this task requires root privileges.

```suggestion
- name: Ensure nix user exists
  become: true
  user:
```

---

I guess. But wondering why `molecule test` passes if this is true?

---

I think that it's because the default value of `nix_user` is `ansible_user`, and the molecule tests [create](https://github.com/Ableton/ansible-role-nix/blob/5bbb3b00f228aecd259ad99ebac4803e6dcba49d/molecule/default/Dockerfile.j2#L11) this user in the Dockerfile, so the task no-ops. We could alter the default scenario to change the value of `nix_user` ([here](https://github.com/Ableton/ansible-role-nix/blob/5bbb3b00f228aecd259ad99ebac4803e6dcba49d/molecule/default/converge.yml#L5)) to ensure this works properly.